src/useLocalStorageState/demo/demo3.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Input } from 'antd';
+import useLocalStorageState from '..';
+
+export default function () {
+  const [user, setUser] = useLocalStorageState('user', {
+    id: 9234634791,
+    name: 'Zhangsan',
+    age: 33,
+  });
+  return (
+    <>
+      <Input
```

Comment:
this `Input` should have a width 200px

![image](https://user-images.githubusercontent.com/12526493/72203245-0f0ccf00-34a4-11ea-804d-05168caab85f.png)

Reply:
done